DungeonGame.py

```diff
@@ -301,15 +306,15 @@ def lava_room():
 
 def infinite_room():
     entries = 0
-    while entries < 10 and entries > 1 and
+    if entries < 10 and entries > 1:
```

Comment:
Unless you build in a `break` or an incrementer to make the condition false, use the `if` here to prevent endless looping.